### DISCUSSION TITLE
plantuml.requarks.io seems to be down

### DISCUSSION THREAD

**Discussion by dominiquec** [requarks/wiki#6080] [Error / Bug Report] [answered]
Hello. plantuml.requarks.io seems to be down again. (Please see date on screenshot.)
![plantuml](https://user-images.githubusercontent.com/1487181/213978160-9e510cf8-c038-4402-b4a8-437925973128.png)

**Comment by iti-marcin-pietrak** [comment]
Hello,
same on my side, both on preview mode and on the rendered pages diagrams defined in the PlantUML (Markdown Page) are not being generated. plantuml.requarks.io server seems to be not responding. Please confirm if this is the right place to raise the issue and to increase the priority of the solution. Thanks

**Comment by NGPixel** [comment, marked as answer]
Service restarted. Should be back now.

**Reply by iti-marcin-pietrak** [reply]
Thank you, diagrams in wiki.js generated by PlantUML are working.

**Comment by dominiquec** [comment]
Thank you very much. We appreciate the work you do for this great piece of
software.
